core/src/main/java/org/openstack4j/model/network/builder/NetQuotaBuilder.java

```diff
@@ -66,4 +66,20 @@
      */
     NetQuotaBuilder securityGroupRule(int securityGroupRule);
 
+
+    /**
+     * See {@link NetQuota#getRbacPolicy()} for details
+     *
+     * @param rbacPolicy
+     * @return NetQuotaBuilder
+     */
+    NetQuotaBuilder rbacPolicy(Integer rbacPolicy);
```

Comment:
Stick to `int` unless there is a justified reason to use `Integer`. We do not seem to need `null` as special value and it will actually fail with currect `NetQuotaConcreteBuilder` impl.

Reply:
You're right. I changed it.